From 94a259311ebb415767b454c5156799f30aee40ad Mon Sep 17 00:00:00 2001
From: Limeng <Meng.Li@windriver.com>
Date: Tue, 18 Dec 2018 16:39:06 +0800
Subject: [PATCH] gpu: mail-400: remove __GFP_COLD flag

According to upstream commit 453f85d43fa9("mm: remove
__GFP_COLD"). Remove __GFP_COLD flag when alloc page.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../driver/src/devicedrv/mali/linux/mali_memory_os_alloc.c              | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/linux/mali_memory_os_alloc.c b/linux/mali_memory_os_alloc.c
index 3379f4f..b1cfb39 100755
--- a/linux/mali_memory_os_alloc.c
+++ b/linux/mali_memory_os_alloc.c
@@ -202,7 +202,7 @@ int mali_mem_os_alloc_pages(mali_mem_os_mem *os_mem, u32 size)
 	/* Allocate new pages, if needed. */
 	for (i = 0; i < remaining; i++) {
 		dma_addr_t dma_addr;
-		gfp_t flags = __GFP_ZERO | __GFP_RETRY_MAYFAIL | __GFP_NOWARN| __GFP_COLD;
+		gfp_t flags = __GFP_ZERO | __GFP_RETRY_MAYFAIL | __GFP_NOWARN;
 		int err;
 
 #if defined(CONFIG_ARM) && !defined(CONFIG_ARM_LPAE)
-- 
2.7.4

